PagesGallery/PagesGallery/PagesGallery.csproj

```diff
@@ -71,6 +71,11 @@
       <Project>{9db2f292-8034-4e06-89ad-98bbda4306b9}</Project>
       <Name>Xamarin.Forms.Xaml</Name>
     </ProjectReference>
+	  <ProjectReference Include="..\..\Xamarin.Forms.Build.Tasks\Xamarin.Forms.Build.Tasks.csproj">
+		  <Project>{96D89208-4EB9-4451-BE73-8A9DF3D9D7B7}</Project>
+		  <Name>Xamarin.Forms.Build.Tasks</Name>
+		  <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+	  </ProjectReference>
```

Comment:
We got this error on our build server... 

 error : Error initializing task FixedCreateCSharpManifestResourceName: Could not load file or assembly '/Applications/buildAgent_c/work/824090adae967c0/.nuspec/Xamarin.Forms.Build.Tasks.dll' or one of its dependencies
[17:38:06][Step 3/6]    		Build FAILED.

adding the reference fixes it